jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -404,6 +404,7 @@ void java_bytecode_convert_method_lazy(
   }
 
   method_symbol.type=member_type;
+  method_symbol.type.set(ID_C_abstract, m.is_abstract);
```

Comment:
This requires a comment as the field is set but never read within the code base -- I assume it is being read in some software outside this repository.